website/content/en/preview/upgrading/v1-migration.md

```diff
@@ -184,11 +184,64 @@ then you have to manually add more EC2NodeClasses and point their NodePools to t
 
 ### Downgrading
-// TODO 
+The patch version of the v1beta1 Karpenter controller that include the conversion wehooks: 
+
+* v0.37.1
+* v0.36.3
+* v0.35.6
+* v0.34.7
+* v0.33.6
+
+{{% alert title="Note" color="warning" %}}
+When rolling back from v1, Karpenter will not retain data that was only valid in v1 APIs. For instance, if you were upgrading from v0.33.5 to v1, and then rolling back to v0.33.6, Karpenter would not retain the `NodePool.Spec.Disruption.Budgets` field, as that was introduced in v0.34.x. If you are configuring the kubelet field, and have removed the `compatibility.karpenter.sh/v1beta1-kubelet-conversion` annotation, rollback is not supported without replacing your nodes between EC2NodeClass and NodePool.
+{{% /alert %}}
+
+{{% alert title="Note" color="warning" %}}
+As both v1beta1 and v1 will be served, `kubectl` will default to returning the `v1` version of your CRDs. To interact with the v1beta1 version of your CRDs, you'll need to add the APIVersion on the end of your `kubectl` calls. For example: `k get nodeclaim.v1beta1.karpenter.sh`
+{{% /alert %}}
+
+Step 1.) Rollback the Karpenter Policy to reflect the version
+
+```bash
+TEMPOUT=$(mktemp)
+curl -fsSL https://raw.githubusercontent.com/aws/karpenter-provider-aws/v"${KARPENTER_VERSION}"/website/content/en/docs/getting-started/getting-started-with-karpenter/cloudformation.yaml > ${TEMPOUT} \
+    && aws cloudformation deploy \
+    --stack-name "Karpenter-${CLUSTER_NAME}" \
+    --template-file "${TEMPOUT}" \
+    --capabilities CAPABILITY_NAMED_IAM \
+    --parameter-overrides "ClusterName=${CLUSTER_NAME}"
+```
+
+Step 2.) Rollback the CRDs to the version that is being rollbacked
+
+```bash
+KARPENTER_NAMESPACE=kube-system 
+helm upgrade --install karpenter-crd oci://public.ecr.aws/karpenter/karpenter-crd --version "${KARPENTER_VERSION}" --namespace "${KARPENTER_NAMESPACE}" --create-namespace \
+  --set webhook.enabled=true \
+  --set webhook.serviceName=karpenter \
+  --set webhook.serviceNamespace="${KARPENTER_NAMESPACE}" \
+  --set webhook.port=8443
+```
+
+Step 3.) Rollback the Karpenter Controller Version 
+```bash
+helm upgrade --install karpenter oci://public.ecr.aws/karpenter/karpenter --version ${KARPENTER_VERSION} --namespace kube-system --create-namespace \
+  --set serviceAccount.annotations."eks\.amazonaws\.com/role-arn"=${KARPENTER_IAM_ROLE_ARN} \
+  --set settings.aws.defaultInstanceProfile=KarpenterNodeInstanceProfile-${CLUSTER_NAME} \
+  --set settings.clusterName=${CLUSTER_NAME} \
+  --set settings.interruptionQueue=${CLUSTER_NAME} \
+  --set controller.resources.requests.cpu=1 \
+  --set controller.resources.requests.memory=1Gi \
+  --set controller.resources.limits.cpu=1 \
+  --set controller.resources.limits.memory=1Gi \
+  --set webhook.enabled=true \
+  --set webhook.port=8443 \
+  --wait
+```
+
```

Comment:
Should we add a bit here at the end that says something like, "Karpenter should now be pulling and operating against the v1beta1 api version as it was prior to the upgrade"